src/browser/components/MainPage.jsx

```diff
@@ -598,6 +602,39 @@ export default class MainPage extends React.Component {
       this.switchToTabForCertificateRequest(certificateRequests[0].server);
     }
   };
+
+  showDownloadCompleteNotification = async (event, item) => {
+    const title = 'Download Complete';
+
+    const notification = await this.dispatchNotification(title, item.title);
+    notification.onclick = () => {
+      shell.showItemInFolder(item.defaultPath.normalize());
+    };
+  }
+
+  dispatchNotification = async (title, body, channel, teamId, silent) => {
```

Comment:
why the teamId? if this is only for download I'd rename to `dispatchDownloadNotification` or something that can be distinguished from the other one.
Should we have all notification code together? I'm 0/5 on this if we don't have a more general case.

Reply:
Agreed, this is repeated code from `MattermostView.jsx`, so if we could find a way to generalize sending notifications between the normal case and the few corner cases I would be in favour of that.

0/5 on if it happens for this PR for not, but if not we should create a ticket for it.